quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/TableDataMigrator.java

```diff
@@ -231,22 +240,22 @@ private Object parseValue(Type type, String value) {
 			case SMALLINT:
 			case INTEGER:
 				return Integer.parseInt(value);
+			case NUMERIC:
```

Comment:
Note that `numeric` can also map to a float or double precision in PG. Those would require a different parser.

Reply:
I have not had any problems with this but I have changed it to the double parser as no information is thrown away:
`return Double.parseDouble(value)`